android/app/src/main/java/im/status/ethereum/MainActivity.java

```diff
@@ -94,13 +94,6 @@ private Intent createNotificationSettingsIntent() {
         return intent;
     }
 
-    @Override
-    public void onNewIntent(final Intent intent) {
-        if (intent.getDataString() != null && intent.getData().getScheme().startsWith("app-settings")) {
-          startActivity(createNotificationSettingsIntent());
```

Comment:
Is this related?

Reply:
this is from the react-native-fcm documentation

Reply:
Well that's not the same thing you removed?

Reply:
@jeluard thanks for the catch, I see this was implemented for easy access to the settings from profile. Thanks, will revert